### ISSUE TITLE
[idea] add option to turn off swipe up to close search

### ISSUE THREAD

**Issue by CDTay** [MM2-0/Kvaesitso#377]
When I'm scrolling through the search results sometimes I keep going for a second after I got the bottom and this closes back to the homescreen. It would be nice if there was an option to just have the overscroll bounce instead of the results closing

**Comment by leleogere** [comment]
I agree with that! I end up having to search something again way too often because of that
